crates/bevy_render/src/camera/camera.rs

```diff
@@ -219,37 +230,72 @@ impl Camera {
     /// The projection matrix computed using this camera's [`CameraProjection`].
     #[inline]
     pub fn projection_matrix(&self) -> Mat4 {
-        self.computed.projection_matrix
+        self.computed.projection_matrix.unwrap_or_default()
```

Comment:
This new default-ing behavior should be listed in the docs.

Reply:
So, the behavior shouldn't really have changed at all here. It's just that instead of the default being an identity matrix that is then served by this function, the default is now None, which this function ignores in favor of serving an identity matrix. To the caller there should be no observable difference.

See: https://github.com/bevyengine/bevy/pull/8841#discussion_r1250898765

Reply:
In other words, there should be no externally observable change here. The change is purely to the benefit of the checked version of this function, which treats the None as an error.